docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md

```diff
@@ -52,12 +52,19 @@ In this step, you will configure the yaml file required for Windows event logs a
 
 Any custom fields can be tagged along with the data in this step.
 
-Once the details are filled in, click on the **Download YAML File** button to get the yaml file.
+#### Enable process metric collection (Optional)
 
-<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Windows-OpenTelemetry/Windows-YAML.png' style={{border:'1px solid black'}} alt="YAML" />
+By default, the collector will not send process metrics to Sumo Logic. This is because the number of processes running on a host can be very large, which would result in a significant increase in Data Points per Minute (DPM).
+
+Click the **Enable process metric collection** checkbox to collect (process level metric)[https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/internal/scraper/processscraper/documentation.md].
+- **Name of process**. Add the list of process names.
+- **Include/Exclude the above pattern**. Signifies if you want to exclude or include the metrics for the processes listed previously.
+- **Match type for process name**. Select if the process name given should be considered for a strict match with the host machine processes or if it should be considered as regex when matching.
+
+<img src='img/integrations/hosts-operating-systems/process-metric-collection.png' alt="process-metric-collection" width="400"/>
```

Comment:
The image is broken because the tagging doesn't include useBaseUrl. For example:
 <img src={useBaseUrl('img/<your-image-file-path>.png')} alt="<your image description>" style={{border: '1px solid black'}} width="<insert-pixel-number>" />

Reply:
fixed!